client/cos.go

```diff
@@ -0,0 +1,261 @@
+// Copyright 2024 OpenPubkey
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
I'm putting this in the files I touch in PR to not overwrite history. At some point we will need to just hit the remaining files.

Reply:
I'm sorry, I'm not sure what you mean by overwrite history. I'm just suggesting that it would make the best sense to do this (since its so important) in its own PR so that it doesn't clutter up this one and we can have better tracking of the changes

Reply:
I see Ethan's point. If we do it later it does kind of pollute the version history of an individual file and make it harder to track a set of changes.

That said, I think until we have a v1 released, anything's fair game, and it would make this PR easier to review if we didn't do it this way. I'd vote to add it to every file at a later date